cypress/e2e/pages/cytAssist.cy.ts

```diff
@@ -8,8 +8,8 @@ describe('CytAssist Page', () => {
   });
 
   describe('On load', () => {
-    it('shows a Visium LP CytAssist slide for the output', () => {
-      cy.findAllByText(/Visium LP CytAssist/i).should('have.length.above', 0);
+    it('output labware type should be empty', () => {
+      cy.findByTestId('output-labware-type').should('have.value', '');
```

Comment:
Would it better to use shouldDisplaySelectedValue and check against the value here?